### ISSUE TITLE
Photos no longer backing up/appearing after upgrade to 1.94.1-ig247

### ISSUE THREAD

**Issue by 3rdStng** [imagegenius/docker-immich#293]
I don't know how to troubleshoot or resolve this. Ever since upgrading to 1.94.1-ig247, photos are no longer backing up or appearing within the app on my phone. If I click on "Recent" or "Camera" for which photos to backup, they appear in the grid view, but they will not appear in the main agenda/date view. Which I think is the reason they are not backing up to the server either. I tried setting my docker back to 1.94.1-ig246 as well as uninstalling and reinstalling the app on my phone. I even logged out and back in, but nothing appears to kick the backups off. Where/How can I troubleshoot this more? 

Docker image - 1.94.1-ig247
Oneplus 12 phone running OOS 14 - app 1.94.1 build 121

This is a new phone so I was thinking back to any changes I made. I did change the Advanced setting to Prefer remote images, since none of the old images from my old phone were showing within the app on my new phone. I just changed this back and increased the logging to Severe. All of the images now show a box with a line through it. Even images that currently reside on my phone. I am still not seeing photos that were taken after the upgrade within the app. I can see them in my native Photos app though.

I am seeing a bunch of these within my logs now.
```
FlutterError - Catch all error:  - PlatformException(The asset not found!, null, null, null) - image resource service - resolving an image codec - #0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:651)
#1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:322)
<asynchronous suspension>
#2      AssetEntityImageProvider._loadAsync.<anonymous closure> (package:photo_manager_image_provider/photo_manager_image_provider.dart:125)
<asynchronous suspension>
#3      AssetEntityImageProvider._loadAsync.<anonymous closure> (package:photo_manager_image_provider/photo_manager_image_provider.dart:150)
<asynchronous suspension>
#4      AssetEntityImageProvider._loadAsync.<anonymous closure> (package:photo_manager_image_provider/photo_manager_image_provider.dart:154)
<asynchronous suspension>
```

**Comment by martabal** [comment]
Sounds like an upstream issue

**Comment by 3rdStng** [comment]
Meaning from the phone to the server? 
I wiped all the cache and storage from the App Info on the phone and basically started over. Making sure I allowed all the permissions within the app before logging into the server. It backed up all the missing photos, skipping the duplicates. I took a few new photos, and those backed up also. But every time I open the app now, it tells me "building the timeline" as if this is my first time logging in. It's done it 3 or 4 times now. Still backing up new photos though. I suppose I could live with the building of the timeline. Just new to me
